src/NuGet.Core/NuGet.CommandLine.XPlat/GlobalSuppressions.cs

```diff
@@ -34,6 +34,7 @@
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.CommandLine.XPlat.MSBuildAPIUtility.AddVersionMetadata(NuGet.LibraryModel.LibraryDependency,Microsoft.Build.Construction.ProjectItemElement)~System.String")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.CommandLine.XPlat.MSBuildAPIUtility.AreCentralVersionRequirementsSatisfied(NuGet.CommandLine.XPlat.PackageReferenceArgs,NuGet.ProjectModel.PackageSpec)~System.Boolean")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.CommandLine.XPlat.MSBuildAPIUtility.CreateItemGroup(Microsoft.Build.Evaluation.Project,System.String)~Microsoft.Build.Construction.ProjectItemGroupElement")]
+[assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.CommandLine.XPlat.MSBuildAPIUtility.GetDirectoryPackagePropsRootElement(Microsoft.Build.Evaluation.Project)~Microsoft.Build.Construction.ProjectRootElement")]
```

Comment:
Please either fix the analyzer violation, or provide a justification. Leaving the "pending" justification was a trade off when analyzers were enabled and there were several thousand (unreasonable to investigate and justify each).

Reply:
I have removed the new suppression I had added

Reply:
Now instead of adding a new suppression, you're modifying an existing one. I think the same principal applies ("boy scout rule" from the Clean Code book), if you modify a suppression, you should either fix it or provide justification.